### ISSUE TITLE
[V2] Problem authenticating from Emby backend

### ISSUE THREAD

**Issue by schmittyd** [causefx/Organizr#1030]
###### Operating System: Ubuntu Server 18.04 LTS x64
<hr>

##### Problem Description:
Can not create/login users from Emby db

<hr>

##### Reproduction Steps:
Login to Organizr with Emby user credentials

<hr>

#### Errors on screen?  If so paste here:

```

15 January 2019 18:47 | schmittyd | 172.16.0.102 | Login Function - Wrong Password |  
-- | -- | -- | -- | --

15 January 2019 18:47 | schmittyd | 172.16.0.102 | Emby Connect Auth Function - Found User |  
-- | -- | -- | -- | --


```

**Comment by schmittyd** [comment]
#894

**Comment by causefx** [comment]
Fixed with 93706ad
